### PR TITLE
Reintroduced Style to /attend

### PR DESCRIPTION
**Pull request by spyhi**
- Added attend.css back in from splash page site
- Added as an import to pages/attend.js
- Fixed a few tags that didn't make transition from HTML to JSX

(Looks like a lot removed an added but it was just adding some enclosing divs and reformatting)

### CODE REVIEW

**Comment by spyhi** [comment]
Forgot to click save on one of my changes before committing. Previous notes still apply.

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "author": "JSConf Hawaii <hello@jsconfhi.com>",
   "dependencies": {
     "babel-plugin-styled-components": "^1.10.6",
+    "eslint-plugin-flowtype": "^4.3.0",
```
Comment (jkup) [inline package.json]: Remove?

**Review by jkup — COMMENTED** [review]
Looks great! I think just remove that npm module and we'll be good to go!

**Comment by spyhi** [comment]
So, I've just realized that I've been adding in grid css tags, which makes it so these pages can't be reused without some refactoring to have the content be agnostic and have grid placement on calling page. I don't anticipate that being an issue in the short term, but figured I'd make note of it in case it was in the works for some reason.

File: package.json (hunk 1 above)
Comment (spyhi) [reply package.json]: Gatsby on Windows didn't want to work with me without it (maybe I'm missing something), but I can remove it.

Comment (jkup) [reply package.json]: What error were you seeing? On which command?

**Review by jkup — APPROVED** [review]
Looks good! Feel free to merge whenever you're done.

File: package.json (hunk 1 above)
Comment (spyhi) [reply package.json]: I get the following error when using `gatsby develop` after `start webpack server` in the init process:
```
Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Failed to load plugin flowtype: Cannot find module 'eslint-plugin-flowtype'
```

Followed by

```
ERROR in ./.cache/app.js
Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Failed to load plugin flowtype: Cannot find module 'eslint-plugin-flowtype'
```

Comment (spyhi) [reply package.json]: For now, I've removed the dependency from the package.json and am just keeping the plugin in my local environment, which allows me to start the gatsby dev server. Happy medium.

**Pull request merged**